ui/js/controllers/perf/dashboard.js

```diff
@@ -22,19 +22,17 @@ perf.controller('dashCtrl', [
         // dashboard customization values
         ['variantDataOpt', 'framework', 'header', 'descP1', 'descP2',
             'linkUrl', 'linkDesc', 'baseTitle', 'variantTitle'].forEach(function (k) {
-                $scope[k] = phDashboardValues[$scope.topic][k];
+                try {
+                    $scope[k] = phDashboardValues[$scope.topic][k];
+                }
+                catch (TypeError) {
+                    $log.debug('"'+ k + '" option not found in ' + '"' + $scope.topic + '" dashboard config');
```

Comment:
We're moving away from `$log` (see [bug 1428508](https://bugzilla.mozilla.org/show_bug.cgi?id=1428508)), so perhaps remove this entirely or else use `console.debug()`?

Reply:
I'll use console.debug() then. Thanks!